<?xml version="1.0" encoding="UTF-8"?>

<DBError statementIndex="0">
  <ErrorMessage>java.sql.SQLSyntaxErrorException: 'NOTIFICATION_DEVICE_ID_2' is not a column in table or VTI 'DEVICE'.</ErrorMessage>
  <Comment> Device [Device]</Comment>
  <Statement> alter table "APP"."DEVICE"   drop column  "NOTIFICATION_DEVICE_ID_2"</Statement>
  <ErrorMessage>java.sql.SQLSyntaxErrorException: 'NOTIFICATION_DEVICE_ID_2' is not a column in table or VTI 'DEVICE'.</ErrorMessage>
  <StackTrace>com.webratio.commons.db.DBException: java.sql.SQLSyntaxErrorException: 'NOTIFICATION_DEVICE_ID_2' is not a column in table or VTI 'DEVICE'.
	at com.webratio.commons.db.utils.DBUtils.execute(DBUtils.java:546)
	at com.webratio.commons.db.utils.DBUtils.executeScriptFiles(DBUtils.java:442)
	at com.webratio.commons.db.utils.DBUtils.main(DBUtils.java:257)
Caused by: java.sql.SQLSyntaxErrorException: 'NOTIFICATION_DEVICE_ID_2' is not a column in table or VTI 'DEVICE'.
	at org.apache.derby.client.am.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.client.am.SqlException.getSQLException(Unknown Source)
	at org.apache.derby.client.am.Statement.execute(Unknown Source)
	at com.webratio.commons.db.utils.DBUtils.execute(DBUtils.java:523)
	... 2 more
Caused by: org.apache.derby.client.am.SqlException: 'NOTIFICATION_DEVICE_ID_2' is not a column in table or VTI 'DEVICE'.
	at org.apache.derby.client.am.Statement.completeSqlca(Unknown Source)
	at org.apache.derby.client.am.Statement.completeExecuteImmediate(Unknown Source)
	at org.apache.derby.client.net.NetStatementReply.parseEXCSQLIMMreply(Unknown Source)
	at org.apache.derby.client.net.NetStatementReply.readExecuteImmediate(Unknown Source)
	at org.apache.derby.client.net.StatementReply.readExecuteImmediate(Unknown Source)
	at org.apache.derby.client.net.NetStatement.readExecuteImmediate_(Unknown Source)
	at org.apache.derby.client.am.Statement.readExecuteImmediate(Unknown Source)
	at org.apache.derby.client.am.Statement.flowExecute(Unknown Source)
	at org.apache.derby.client.am.Statement.executeX(Unknown Source)
	... 4 more
</StackTrace></DBError>